CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ CHANGELOG
   to support storage retention on deletion.
 - Enable server-side encryption for the EcrImageBuilder SNS topic created when deploying ParallelCluster API and used to notify on docker image build events.
 - Add support for on-demand capacity reservations.
+- Add support for requiring IMDSv2 in cluster and build image configurations.
```

Comment:
I think we should mention the specific property: `Imds > RequireImdsV2 `

Reply:
I am okay with this change, however note that we didn't do this for the immediately preceding item `Add support for on-demand capacity reservations.`

Reply:
Changed in the last revision.